test/reflection_test.cpp

```diff
@@ -42,51 +61,192 @@ struct NonConstexprDefaultConstructibleWithFields
     }
 };
 
+constexpr std::string_view pick_compiler_specific_string([[maybe_unused]] const std::string_view s1,
+                                                         [[maybe_unused]] const std::string_view s2)
+{
+#if defined(__clang__) && __clang_major__ == 15
+    return s1;
+#else
+    return s2;
+#endif
+}
+
 }  // namespace
 
+TEST(Reflection, DebuggingHelper)
+{
+    using enum reflection_detail::RecursionType;
+    auto foo = reflection_detail::field_info_of<RECURSIVE, MyColors>();
+    // std::cout << foo.size() << std::endl;
```

Comment:
Did you mean to leave this commented-out line here?

Reply:
Yeah, this function is for quickly investigating. I had not committed it until know, but after writing it for the n-th time, I left in a small stub in for the future.